Switch-Component (#2846)
cerner/terra-core: packages/terra-switch/src/terra-dev-site/doc/switch/Switch.1.doc.mdx

import { Badge } from 'terra-switch/package.json?dev-site-package';

import DefaultSwitch from '../example/DefaultSwitch?dev-site-example';
import SwitchDisabled from '../example/SwitchDisabled?dev-site-example';
import SwitchLongTextLabel from '../example/SwitchLongTextLabel?dev-site-example';

import SwitchPropsTable from 'terra-switch/src/Switch?dev-site-props-table';

> 

Reviewer: Looks like you forgot the `badge` here.

Author: Added in this [commit](https://github.com/cerner/terra-core/pull/2846/commits/1c002b1da07d995fc126d14539617437b6371dd7)

import { Badge } from 'terra-switch/package.json?dev-site-package';

import DefaultSwitch from '../example/DefaultSwitch?dev-site-example';
import SwitchWithOnChange from '../example/SwitchWithOnChange?dev-site-example';
import DisabledSwitch from '../example/DisabledSwitch?dev-site-example';
import SwitchWithLongLabelText from '../example/SwitchWithLongLabelText?dev-site-example';

import SwitchPropsTable from 'terra-switch/src/Switch?dev-site-props-table';

<Badge />

# Terra Switch

> The terra-switch component are like their real-world counterparts, are used to indicate whether a setting is on or off and provide users with instantaneous feedback. 

Reviewer 2: ```suggestion
The terra-switch component is, like its real-world counterpart, is used to indicate whether a setting is on or off and provide users with instantaneous feedback. 
```

Reviewer 3: Lets keep the documentation/description of terra-switch consistent by making the same change in the "README.md" file.

Author: updated with commit : d8ea53ba069375ab4efb9abebf1bd67951aa067f

> The terra-switch component are like their real-world counterparts, are used to indicate whether a setting is on or off and provide users with instantaneous feedback. 
> 
> Switches are used to activate settings immediately, in real time. 
> Switches never require users to press a button to apply their settings. 
> A switch is a set of binary options and always has one of the options selected by default.
> 
> Switch should be used for only one of these set of binary options:
> • On / Off

Reviewer 3: ```suggestion
The terra-switch component indicates whether a setting is ON or OFF and provide users with instantaneous feedback.
```

Author: updated : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864

> Switch should be used for only one of these set of binary options:
> • On / Off

Reviewer 2: I don't think these lines are needed. Consumers can't change the on/off text anyway.
```suggestion
```

Author: removed this too with commit : d8ea53ba069375ab4efb9abebf1bd67951aa067f

## Getting Started

- Install with [npmjs](https://www.npmjs.com):
  - `npm install terra-switch`

<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
## Peer Dependencies

This component requires the following peer dependencies be installed in your app for the component to properly function.

| Peer Dependency | Version |
|-|-|
| react | ^16.8.5 |
| react-dom | ^16.8.5 |

<!-- AUTO-GENERATED-CONTENT:END -->

> ## Usage
> 
> ```jsx
> import Switch from 'terra-switch';
> ```

Reviewer 2: Usage section isn't really needed, the examples all show this.
```suggestion
```

Reviewer 3: Although this might not be needed, but to keep things consistent it would be better if this was kept.
During the MDX updates, it was decided to add these to all the packages.
